Documentation deploy is broken

When releasing 1.0.11 (refs https://github.com/lightbend/kalix-jvm-sdk/issues/1249) the publish-docs broke with:
```shell
error: missing required argument: upstream
make: *** [Makefile:115: deploy] Error 1
```

I would guess this might be related with some changes we did for fitting in Spring docs but couldn't spot any obvious problem right away.

See pipeline [here](https://app.circleci.com/pipelines/github/lightbend/kalix-jvm-sdk/4803/workflows/0ca35637-ebe9-48b7-8f57-4271dba651a8/jobs/81414).

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -112,5 +112,6 @@ validate-links:
 		-t ${antora_docker_image}:${antora_docker_image_tag} \
 		-c "cd /antora/${base_path} && find src -name '*.adoc' -print0 | xargs -0 -n1 asciidoc-link-check --progress --config config/validate-links.json"
 
+# TODO: we will need to include publication for spring module once we want it out
 deploy: clean managed
-	bin/deploy.sh --module ${module} --upstream ${upstream} --branch ${branch} ${sources}
+	bin/deploy.sh --module java --upstream ${upstream} --branch ${branch} ${sources}

--- a/docs/bin/deploy.sh
+++ b/docs/bin/deploy.sh
@@ -15,7 +15,7 @@
 # deploy author
 
 readonly deploy_name="Kalix Bot"
-readonly deploy_email="bot@kalix.io"
+readonly deploy_email="kalix.github@lightbend.com"
 
 # locations
 

